src/databases/Xolotl/XolotlCommonPluginInfo.C

```diff
@@ -0,0 +1,144 @@
+/*****************************************************************************
```

Comment:
The generated ".h" and ".C" files all have our old copyright notice block. We just changed it to be much smaller and probably only have the code generation tools updated on develop, which explains why you got the old copyright notice block. Can you replace it with:

// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
// Project developers.  See the top-level LICENSE file for dates and other
// details.  No copyright assignment is required to contribute to VisIt.

Reply:
Absolutely, I have updated all of the files to the correct copyright.